components/actionbar/stories/template.js

```diff
@@ -18,67 +18,56 @@ export const Template = ({
 	isFlexible = false,
 	customClasses = [],
 	...globals
-}) => {
```

Comment:
The themes files were empty so they were removed & don't need to be loaded in the storybook templates.